src/index.html

```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--start-day--sat calendar--month-length--29">
+      <div class="calendar__content">
```

Comment:
You do not need to create additional `div` with the `calendar__content` class, it is enough to style the `calendar` class correctly.

Reply:
I don't know how to display a calendar in the middle of the screen while not creating additional div

Reply:
@ivannaromanovych You need to set the necessary styles for the calendar class.